x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel.tsx

```diff
@@ -693,6 +697,12 @@ export const VisualizationWrapper = ({
   // Used for reporting
   const { isRenderComplete, hasDynamicError, setIsRenderComplete, setDynamicError, nodeRef } =
     useReportingState(errors);
+
+  const onRenderHandler = useCallback(() => {
```

Comment:
important to memoize since the inline version was causing multiple subscriptions to be created in the expression renderer